components/infobox/commons/infobox_basic.lua

```diff
@@ -0,0 +1,50 @@
+local Class = require('Module:Class')
+local Infobox = require('Module:Infobox')
+local getArgs = require('Module:Arguments').getArgs
+
+local BasicInfobox = Class.new(
+	function(self, frame)
+		local args = getArgs(frame)
+		self.frame = frame
+		self.pagename = mw.title.getCurrentTitle().title
+		self.name = args.name or self.pagename
+
+		if args.wiki == nil then
+			return error('Please provide a wiki!')
+		end
+
+		self.infobox = Infobox:create(frame, args.wiki)
+	end
+)
+
+--- Allows for overriding this functionality
+function BasicInfobox:getDisplayName(args)
+	return args.name
+end
+
+--- Allows for overriding this functionality
+function BasicInfobox:addCustomCells(infobox, args)
+    return infobox
+end
+
+--- Allows for overriding this functionality
+function BasicInfobox:addCustomContent(infobox, args)
+    return infobox
+end
+
+--- Allows for overriding this functionality
+function BasicInfobox:createBottomContent(infobox)
+    return infobox
+end
+
+--- Allows for overriding this functionality
+function BasicInfobox:addBaseDisplay(infobox, informationType, args)
+	infobox:name(BasicInfobox:getDisplayName(args))
+	infobox:image(args.image, args.defaultImage)
+	infobox:centeredCell(args.caption)
+	infobox:header(informationType .. ' Information', true)
+
+	return infobox
+end
```

Comment:
I think this function is too specific and we should leave it out

Reply:
i mean we use this in all infoboxes, right?
it is always the same code (except for the stuff that is adjustable via `informationType` and `getDisplayName` )

Reply:
It's the same code except often it isn't, it saves too little code and needs too much adjustment for it to be worth it for me

Reply:
okay, will kick it